src/httpserver.cpp

```diff
@@ -563,8 +588,7 @@ std::pair<bool, std::string> HTTPRequest::GetHeader(const std::string& hdr) cons
 std::string HTTPRequest::ReadBody()
 {
     struct evbuffer* buf = evhttp_request_get_input_buffer(req);
-    if (!buf)
-        return "";
+    if (!buf) return "";
```

Comment:
Remove this unrelated whitespace change.

Reply:
yes, that was something I touched while I was verifying the `req` class member.